mrmustard/physics/triples.py

```diff
@@ -218,6 +220,36 @@ def two_mode_squeezed_vacuum_state_Abc(
     return A, b, c
 
 
+def sauron_state_Abc(n: int, epsilon: float):
+    r"""
+    The A,b,c parametrization of Sauron states. These are Fock states written as a linear superposition of a
+    ring of coherent states.
+
+    Args:
+        n: number of photons
+        epsilon: size of the ring. The approximation is exact in the limit for epsilon that goes to zero
+
+    Returns:
+        The ``(A, b, c)`` triple of the sauron state.
+    """
+
+    phases = np.linspace(0, 2 * np.pi * (1 - 1 / (n + 1)), n + 1)
```

Comment:
These `np` calls should be `math` right?

Reply:
no need: it's not about a differentiable variable